app/models/user.rb

```diff
@@ -1,3 +1,4 @@
+# rubocop:disable all
 class User < ApplicationRecord
   # Include default devise modules. Others available are:
```

Comment:
Disabled linters
- Remove statement that disables linters, because it's not good practice to disable them